printer_mods/jbisson9/Backside_Cable_Management/Readme.md

```diff
@@ -0,0 +1,53 @@
+
+# BackSide Cable Management
+
+This is alternative wiring management solution to current methods.  And it was fun to design :)  The idea was to terminate cable within the channel with microfit connectors for easy maintenance.  Guides are used underneath to secure the wiring.  An optional screw and heat set insert are used to secure the system in place if needed.  Currently designed for V2.4 350mm.  CAD will need modification to work with other size and types of printers.
+
+<img src="Images/Main_design.png" width="400px" />
+<img src="Images/complete_system.jpg" width="400px" />
+
+Note microfit connectors are pressure fit with specials retention jigs.
+
+<img src="Images/Guides_and_Pressure_fit_connections.png" width="400px" />
+<img src="Images/view_from_fan.jpg" width="400px" />
+
+The BTT smart filament sensor fits into the printed part.  This is a remix from an unknown design listed on teamfdm.com (thanks!)
+
+<img src="Images/view_from_BTT.jpg" width="400px" />
+
+The system makes use of the keystone skirt, normally for USB/Rasp Pi connections, but in this case two microfit connectors are pressure fit into the slots
+
+<img src="Images/view_from_base.jpg" width="400px" />
+
+# BOM
+* 4x M3 x 12 SHCS
+* 4x M3 Hammerhead T-Nuts
+* BTT filament sensor (should come with 2x M3 x 16 BHCS)
+* 2-, 3- and 5-circuit microfit 3 connectors
+* Optional M3 heat set insert and M3 x 12 SCHS
+
+# Printing
+
+Print what you need.  Uses standard Voron print profiles.  Supports not needed but in reality the BTT jig could use it (sorry first design and I don't have all my sh*^Y#t together)
+
+# Assembly
+
+Attach the BTT sensor to the jig with two M3 button head screws.  Create the wiring harnesses to attach the microfit connectors.  As an aside, all three ends are males connectors, unusual generally, but easier to build jigs around them.  
```

Comment:
These instructions are a bit unclear.  There are no stls with 'jig' anywhere in the filename. I'm not sure what part 'the jig' refers to -- `microfit_jig.jpg` and `completed_jig.jpg` both show parts that don't have any obvious way to connect to the btt sensor, is this a different jig?

Reply:
@MapleLeafMakers going to rename these images and assembly to clam shells.